guides/v2.2/javascript-dev-guide/widgets/widget-breadcrumbs.md

```diff
@@ -0,0 +1,156 @@
+title: Breadcrumbs widget
+contributor_name: Atwix
+contributor_link: https://www.atwix.com/
+---
+
+## Overview
+
+The Breadcrumbs [widget](https://glossary.magento.com/widget/) allows building a list of links which should show where you are in relation to other pages in the store.
+
+The Breadcrumbs widget can be used only on the frontend area.
+
+The widget source file is [`<Magento_Theme_module_dir>/view/frontend/web/js/view/breadcrumbs.js`][]
+
+The Breadcrumbs widget template source file is [`<Magento_Theme_module_dir>/view/frontend/web/templates/breadcrumbs.html`][]
+
+## Initialize the Breadcrumbs widget
+
+For information about how to initialize a widget in a JS component or `.phtml` template, see the [Initialize JavaScript][] topic.
+
+Generally the Breadcrumbs widget is instantiated like following:
+
+```javascript
+$('#breadcrumbs').breadcrumbs();
+```
+
+Where:
+
+- `#breadcrumbs` is the selector of the block element for which Breadcrumbs is initialized.
+
+Phtml template file examples using script:
+
+```html
+<script>
+    require([
+            'jquery',
+            'breadcrumbs'
+        ], function ($) {
+            'use strict';
+    
+            $('#breadcrumbs').breadcrumbs();
+        });
+</script>
+```
+
+## Breadcrumb List
+
+The Breadcrumb List is used to configure the breadcrumbs list.
+
+The Breadcrumb List represents a simple array which stores breadcrumbs items.
+
+The Breadcrumb List source file is [`<Magento_Theme_module_dir>/view/frontend/web/js/model/breadcrumb-list.js`][]
+
+The Breadcrumb widget uses the Breadcrumbs List to configure a list of breadcrumbs which should be rendered. 
+
+The Breadcrumb List should be configured before Breadcrumbs widget initialization. After Breadcrumbs widget initialization the Breadcrumb List does not affect the rendered list of breadcrumbs.
+
+Code example of adding a new breadcrumb item to the Breadcrumb List:
+
+```html
+<script>
+    require([
+        'Magento_Theme/js/model/breadcrumb-list'
+    ], function (breadcrumbList) {
+        'use strict';
+
+        var breadcrumbs = [
+            {
+                'name': 'some-page-1',
+                'link': '/some/path/1',
+                'title': 'Link title for page 1',
+                'label': 'Some page 1'
+            },
+            {
+                'name': 'some-page-1',
+                'link': '/some/path/2',
+                'title': 'Link title for page 1',
+                'label': 'Some page 2'
+            }
+        ];
+
+        breadcrumbs.forEach(function (breadcrumb) {
+            breadcrumbList.push(breadcrumb);
+        });
+    });
+</script>
+```
+
+### Breadcrumb List Item Options
+
+The Breadcrumb List Items should have the following structure:
+
+| Option | Description | Type |
+| --- | --- | --- |
+| `name` | The class of a breadcrumb item. | String |
+| `link` | The URL link of a breadcrumb item. | String |
+| `title` | The title of a breadcrumb item link. | String |
```

Comment:
Hi @serhiyzhovnir. Thank you for the great article! You've described everything well, but the point with list item title and label might not be obvious. The question that appears, where exactly title|label is being shown? Since `<a>` tag does not have a native `label` property, it's hard to guess what's the difference in title|label.

Thank you